src/main/java/com/khomishchak/ws/controllers/BalanceController.java

```diff
@@ -0,0 +1,27 @@
+package com.khomishchak.ws.controllers;
+
+import com.khomishchak.ws.services.exchangers.ExchangerService;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.time.LocalDateTime;
+
+@RestController
+@RequestMapping("/api/v1/balances")
+public class BalanceController {
+
+    private final ExchangerService exchangerService;
+
+    public BalanceController(ExchangerService exchangerService) {
+        this.exchangerService = exchangerService;
+    }
+
+    @GetMapping("/transactions-history/period")
+    public Double getDepositAmountForPeriod(@RequestParam Long userId, @RequestParam String ticker,
```

Comment:
It could be more flexible if you make it possible to not bound itself by providing date range as required parameter. The reasons to bound API like this one are:
1) You have a big amount of data 
2) You have sensitive data
I think it's not a case of any. So you can move it to simple `/transactions/history` endpoint. Set date range as optional parameters. It will make your API more flexible.